### ISSUE TITLE
plugin gen code use `const` not `var`

### ISSUE THREAD

**Issue by HighSkySky** [eight04/rollup-plugin-external-globals#41]
In the latest version of rollup-plugin-external-globals, using the following configuration:

```ts
  externalGlobals({
    'micro-app': 'window.MicroApp',
  }),
```

The original code:
```ts
export { navigateTo, start } from 'micro-app';
Will generate output containing the const keyword:
```

```ts
const _global_window_MicroApp_navigateTo = window.MicroApp.navigateTo;
const _global_window_MicroApp_start = window.MicroApp.start;
```
My code was written in TypeScript and transpiled to ES5, but the const generated by this plugin is the only piece of ES6 syntax.
I need to ensure that there is no ES6 syntax in my project.

**Comment by HighSkySky** [comment]
I have reviewed the Rollup documentation, and internally Rollup uses [constBindings](https://cn.rollupjs.org/configuration-options/#output-generatedcode-constbindings) to control whether the generated code uses const or var (the default is var). I think the behavior of this plugin could also follow this configuration option.

Additionally, I could actually solve this problem by using Babel, but I don't want to use Babel just for this one const.

**Comment by eight04** [comment]
Fixed in #48
